### DISCUSSION TITLE
License issue

### DISCUSSION THREAD

**Discussion by robdekort** [teamnovu/statamic-images-missing-alt#5] [General]
Hey there,

I'm the author of Peak. Thanks for extracting the widget, enhancing it and making it available for free for non Peak users. I really support that. I took your caching strategy and merged it into Peak this week. Great call on that.

However, you're in breach of the Peak license. This addon should be released under the GPL license just like Peak itself. 

This prevents folks from releasing parts of Peak paid. 

Would you please remedy this?

All the best.

Rob

**Comment by robdekort** [comment]
I'd appreciate a response on this.

**Comment by buffalom** [comment]
Hey Rob,

Thanks for reaching out, and apologies for the slow response.
Also, thank you for your support and for adopting the caching strategy. I'll give at least some of the credit to @Kylewalow. 😄 

I just changed the licence to GPL. Let me know if there's anything else I can help you with.

Best,
Tom

**Comment by robdekort** [comment]
Brilliant. Thank you!
